app-backend/batch/src/main/scala/com/azavea/rf/batch/stac/ReadStacFeature.scala

```diff
@@ -1,11 +1,236 @@
+
+    opt[Unit]('t', "test")
+      .action(
+        (_, conf) =>
+        conf.copy(testRun = true)
+      ).text("Dry run stac conversion to scene to verify output")
+
+    opt[String]('p', "path")
+      .required()
+      .action(
+        (p, conf) =>
+        conf.copy(path = p)
+      ).text("STAC geojson URI. ex: 'file:///opt/raster-foundry/app-backend/stac.geojson' 's3://{uri}' 'http://{uri}'")
+
+    opt[String]('d', "datasource")
+      .required()
+      .action(
+        (d, conf) => {
+          conf.copy(datasource = UUID.fromString(d))
+        }
+      ).text("Datasource to create scene with")
+  }
+}
+
+object ReadStacFeature extends Config with LazyLogging {
   val name = "read_stac_feature"
   def main(args: Array[String]): Unit = {
-    val allArgs = args.mkString(" ")
-    logger.info(s"Read Stac Feature operation called with args: ${allArgs}")
+    implicit val db = DB.DEFAULT
+    val params = CommandLine.parser.parse(args, CommandLine.Params()) match {
+      case Some(params) =>
+        params
+      case None =>
+        return
+    }
+    val geojson = Source.fromInputStream(getStream(new URI(params.path))).getLines.mkString
+    val decoded = decode[stac.Feature](geojson)
+    decoded match {
+      case Right(stacFeature) =>
+        val scene = stacFeatureToScene(stacFeature, params.datasource, params)
+        params.testRun match {
+          case true =>
+            logger.info(s"Test run, so scene was not actually created:\n${scene}")
+          case _ =>
+            Await.result(writeSceneToDb(scene), 5 seconds)
+        }
+      case Left(error) =>
+        logger.error(s"There was an error decoding the geojson into a stac Feature: ${error.getLocalizedMessage}")
+    }
+  }
+
+  protected def stacFeatureToScene(
+    feature: stac.Feature,
+    datasource: UUID,
+    params: CommandLine.Params
+  ): Scene.Create = {
+    val thumbnailLinks = feature.links.filter(_.`type` == "thumbnail")
+
+
+    // if datasource is defined, use bands from datasource
+    val (imageAssets, metadataFiles) = feature.assets.partition(asset => asset.format.getOrElse("none") == "tif")
+
+    val sceneId = UUID.randomUUID()
+    val images = getBandedImages(imageAssets, sceneId) // get bands from the image products
+
+    val srid = feature.geometry.srid
+
+    val geom = feature.geometry.geom.reproject(CRS.fromEpsgCode(srid), CRS.fromEpsgCode(3857))
+    val dataFootprint = geom.as[Polygon].map(g => Projected(MultiPolygon(g), 3857)).orElse(geom.as[MultiPolygon].map(g => Projected(g, 3857)))
+
+    Scene.Create(
+      id = Some(sceneId),
+      organizationId = landsat8Config.organizationUUID, // this config should be changed to a generic import config
+      ingestSizeBytes = 0,
+      visibility = Visibility.Public,
+      tags = feature.properties.provider.split(",").map(_.trim).toList,
+      datasource = datasource,
+      sceneMetadata = Json.Null, // stac feature fields that don't fit on the scene
```

Comment:
I think since we don't explicitly store start/end we should make sure they end up here